src/build/help/help.xml

```diff
@@ -506,6 +506,21 @@
                         <example>path</example>
                     </config-key>
 
+                    <config-key id="repo-block" name="Block Incremental Backup">
+                        <summary>Enable block incremental backup.</summary>
+
+                        <text>
+                            <p>Block incremental allows for more granular backups by splitting files into blocks that can be backed up independently. This saves space in the repository and can improve delta restore performance because individual blocks can be fetched without reading the entire file from the repository.</p>
+
+                            <admonition type="note">The <br-option>repo-bundle</br-option> option must be enabled before <br-option>repo-block</br-option> can be enabled.</admonition>
+
+                            <p>The block size for a file is determined based on the file size and age. Generally, older/larger files will get larger block sizes. If a file is old enough, it will not be backed up using block incremental.</p>
+
+                            <p>Block incremental is most efficient when enabled for all backup types, including full. This makes the full a bit larger but subsequent differential and incremental backups can make use of the block maps generated by the full backup to save space.</p>
```

Comment:
Wouldn't it be worth it to force to have repo-block enabled for all backup types? Just as repo-bundle must also be enabled...

Reply:
We could do that, but I would like the user to be able to enable this without needing to wait until the next full backup. Even when enabled not all files get a block incremental (depending on size/age) so all block incremental backups are hybrids and it does not matter if it is turned on an off.

Also, we can't just force it as the default because some users depend on being able to snapshot and use the backups as-is.

Reply:
OK, seems reasonable then :+1: